pandas/tests/plotting/test_datetimelike.py

```diff
@@ -815,6 +815,23 @@ def test_mixed_freq_shared_ax(self):
         # self.assertEqual(ax1.lines[0].get_xydata()[0, 0],
         #                  ax2.lines[0].get_xydata()[0, 0])
 
+    def test_nat_handling(self):
+
+        import matplotlib.pyplot as plt  # noqa
+
+        fig = plt.gcf()
+        plt.clf()
+        ax = fig.add_subplot(111)
+
+        dti = DatetimeIndex(['2015-01-01', NaT, '2015-01-03'])
+        s = Series(range(len(dti)), dti)
+        s.plot(ax=ax)
+        xdata = ax.get_lines()[0].get_xdata()
+        # plot x data is bounded by index values
+        self.assertLessEqual(s.index.min(), Series(xdata).min())
```

Comment:
We just switched over to pytest, so this can be `assert s.index.min() <= Series(xdata).min()`. Same thing with the next line.